src/app/(main)/chat/@session/features/SessionListContent/Inbox/index.tsx

```diff
@@ -5,21 +5,24 @@ import { useTranslation } from 'react-i18next';
 import { DEFAULT_INBOX_AVATAR } from '@/const/meta';
 import { INBOX_SESSION_ID } from '@/const/session';
 import { SESSION_CHAT_URL } from '@/const/url';
-import { useServerConfigStore } from '@/store/serverConfig';
+import { featureFlagsSelectors, useServerConfigStore } from '@/store/serverConfig';
 import { useSessionStore } from '@/store/session';
 
 import ListItem from '../ListItem';
 import { useSwitchSession } from '../useSwitchSession';
 
 const Inbox = memo(() => {
-  const { t } = useTranslation('chat');
+  const { t } = useTranslation(['chat', 'custom']);
   const mobile = useServerConfigStore((s) => s.isMobile);
   const activeId = useSessionStore((s) => s.activeId);
   const switchSession = useSwitchSession();
-
+  const { enableCommercialInbox } = useServerConfigStore(featureFlagsSelectors);
```

Comment:
as there a multi-time repeat the inboxDisplayTitle, I think it's better to create a hook for InboxMeta, like 

```ts
const { title, desc, avatar }= useInboxAgentMeta();
```

and also need to create a UT for this hooks